src/vivarium/framework/utilities.py

```diff
@@ -171,3 +171,10 @@ def wrapped(*args: Any, **kwargs: Any) -> Any:
                 raise
 
     return wrapped
+
+
+def combine_queries(query1: str | list[str], query2: str | list[str]) -> str:
```

Comment:
What if this method could take any number of queries and joined them? Couldn't it be like this?
```python
def combine_queries(*queries: str) -> str:
    return " and ".join(filter(None, queries))
```
I also think that this still most naturally fits in pop_utils like you had it before, despite the fact that it is now used in the results system, because these queries are ultimately still population queries.